services/minespace-web/src/utils/Validate.js

```diff
@@ -83,3 +85,6 @@ export const validateStartDate = memoize((previousStartDate) => (value) =>
     ? "New manager's start date cannot be on or before the previous manager's start date."
     : undefined
 );
+
+export const yearNotInFuture = (value) =>
+  value && value > moment().year() ? "Year cannot be in future" : undefined;
```

Comment:
I thought utils were in the common folders?

Reply:
Correction, it is. Can you remove this and update the file to use yearNotInFuture form the common folder?

Reply:
MineSpace still has its own Validate file in src. I could move this function into the common folder if you like

Reply:
It should already exist in the common folder!

Reply:
Nice, I looked but I didn't see it the first time. Using it now.